services/nss-coverage/launch-worker.sh

```diff
@@ -62,41 +56,52 @@ if [[ ! -d nss ]]; then
   retry hg clone -r "$NSS_TAG" https://hg.mozilla.org/projects/nss
 fi
 
-# download corpus
-update-ec2-status "[$(date -Iseconds)] setup: downloading corpus"
-mkdir -p corpus
-cd corpus
-tls_targets=()
-non_tls_targets=()
-for p in ../nss/fuzz/options/*; do
-  fuzzer="$(basename "$p" .options)"
-  if [[ "$fuzzer" = "${fuzzer%-no_fuzzer_mode}" ]]; then
-    tls_targets+=("$fuzzer")
-  else
-    non_tls_targets+=("${fuzzer%-no_fuzzer_mode}")
-  fi
-  if [[ ! -d "$fuzzer" ]]; then
-    retry-curl -O "https://storage.googleapis.com/nss-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/nss_$fuzzer/public.zip"
-    mkdir "$fuzzer"
-    cd "$fuzzer"
-    unzip ../public.zip
-    cd ..
+# Clone cryptofuzz
+update-ec2-status "[$(date -Iseconds)] setup: cloning cryptofuzz"
+if [[ ! -d cryptofuzz ]]; then
+  retry git clone --depth 1 https://github.com/guidovranken/cryptofuzz.git
```

Comment:
```suggestion
  git-clone https://github.com/guidovranken/cryptofuzz.git
```

Reply:
`git-clone` is defined in common.sh and works around problems that happen when network glitches happen (eg.  if retry `git clone`, the second run will always fail because the destination folder exists). it already does a shallow clone for you too.